documents/specs/Documentation.md

```diff
@@ -0,0 +1,104 @@
+# Element Documentation
```

Comment:
why does this file have a capital `D` in its name? It goes against unwritten convention